svthalia/concrexit
# Allow SVG slides

Closes #1385

### Summary
Allows slides to be uploaded in the svg format

### How to test
Steps to test the changes you made:
1. Go to slides admin
2. Add slide with an svg file
3. Check the slides to see if the svg is displayed correctly
